### ISSUE TITLE
In case of SQLite3 database on Mac, it doesn't work

### ISSUE THREAD

**Issue by corselia** [sue445/plant_erd#48]
On macOS, when the database is `SQLite3`, an error occurs as below.

```bash
$ ./plant_erd sqlite3 --database development.sqlite3
2019/12/17 09:45:05 Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
```

So I built on Ubuntu with `CGO_ENABLED=1` but an error message was shown.

```bash
$ GOOS=darwin GOARCH=amd64 CGO_ENABLED=1 go build
# os/user
../../../../.goenv/versions/1.13.0/src/os/user/getgrouplist_darwin.go: In function ‘mygetgrouplist’:
../../../../.goenv/versions/1.13.0/src/os/user/getgrouplist_darwin.go:16:11: warning: implicit declaration of function ‘getgrouplist’; did you mean ‘mygetgrouplist’? [-Wimplicit-function-declaration]
  int rv = getgrouplist(user, (int) group, buf, ngroups);
           ^~~~~~~~~~~~
           mygetgrouplist
# net
../../../../.goenv/versions/1.13.0/src/net/cgo_bsd.go:15:72: could not determine kind of name for C.AI_MASK
```

I suppose that though a cross-compile is required to build `go-sqlite3` for darwin, Ubuntu cannot do cross-compile for darwin (or doing so is very difficult).

# References
I referenced below articles.

- [Cross compiling darwin from linux · Issue #33 · mozilla/masche](https://github.com/mozilla/masche/issues/33)
- [Cross-compiling for Darwin amd64 fails on Linux · Issue #22510 · golang/go](https://github.com/golang/go/issues/22510)
- [macos - issue while creating go build for Mac from linux? - Stack Overflow](https://stackoverflow.com/questions/54435099/issue-while-creating-go-build-for-mac-from-linux/54449932)
- [linux - Cross-compilation of cgo (for darwin) fails - Stack Overflow](https://stackoverflow.com/questions/34084597/cross-compilation-of-cgo-for-darwin-fails/34506947)
- [CGO_ENABLED=0 breaks go-sqlite3, Error : Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work · Issue #52 · google/ko](https://github.com/google/ko/issues/52)
- [Cross compiling fails · Issue #384 · mattn/go-sqlite3](https://github.com/mattn/go-sqlite3/issues/384)
- [tpoechtrager/osxcross: Mac OS X cross toolchain for Linux, FreeBSD, OpenBSD and Android (Termux)](https://github.com/tpoechtrager/osxcross)

# Note
- When I built on macOS, it works correcctly

# Environment

#### macOS
```
Darwin XXX 19.0.0 Darwin Kernel Version 19.0.0: Thu Oct 17 16:17:15 PDT 2019; root:xnu-6153.41.3~29/RELEASE_X86_64 x86_64
```

#### Ubuntu
```
Linux XXX 5.0.0-36-generic #39~18.04.1-Ubuntu SMP Tue Nov 12 11:09:50 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by sue445** [comment]
@corselia Thank you for reporting.

I fixed at #49 and released v0.1.1.beta1.

Please check this.
https://github.com/sue445/plant_erd/releases/tag/v0.1.1.beta1

**Comment by corselia** [comment]
@sue445 
Thank you for early reply!
I confirmed it was fixed. I appreciate it.

And congratulations the debut in `Software Design` 🎉

**Comment by sue445** [comment]
Released v0.1.1 (same to v0.1.1.beta1)
